lib/tiddle/strategy.rb

```diff
@@ -43,6 +43,10 @@ def model_name
           Tiddle::ModelName.new.with_underscores(mapping.to)
         end
 
+        def authentication_keys
+          Devise.authentication_keys
```

Comment:
I believe that we should use ```mapping.to.authentication_keys``` as in https://github.com/plataformatec/devise/blob/1ac4bdd09012136432db7d3098c3b0d4f49cd720/lib/devise/strategies/authenticatable.rb#L135

That's because ```authentication_keys``` may be specified on per-model basis http://stackoverflow.com/questions/4633077/how-to-use-different-authentication-keys-for-two-devise-models